docs/adding_tracks.rst

```diff
@@ -6,8 +6,43 @@ Definition
 
 A track describes one or more benchmarking scenarios. Its structure is described in detail in the :doc:`track reference </track>`.
 
-Example track
--------------
+.. _add_track_create_track:
+
+Creating a track from data in an existing cluster
+-------------------------------------------------
+
+If you already have a cluster with data in it you can use the ``create-track`` subcommand of Rally to create a basic Rally track. To create a Rally track with data from the indices ``products`` and ``companies`` that are hosted by a locally running Elasticsearch cluster, issue the following command::
+
+    esrally create-track --track=acme --target-hosts=127.0.0.1: --indices="products,companies" --output-path=~/tracks
```

Comment:
The port is missing here?

Reply:
D'oh! Added again in https://github.com/elastic/rally/pull/970/commits/4419794f76398eacfe4aad2eff7e00c46be7841a.